### PR TITLE
Update to AT telemetry for FOCUSZ

### PR DESCRIPTION
**Pull request by menanteau**

### CODE REVIEW

File: etc/conf/atTelemetry.yaml [hunk 1]
```diff
@@ -55,6 +52,7 @@ end_collection_event:
 imageParam_event:
     device: ATCamera
     topic: imageReadoutParameters
+    array_keys: ccdNames
```
Comment (patrickingraham) [inline etc/conf/atTelemetry.yaml:55]: what's this??

File: etc/conf/ccTelemetry.yaml [hunk 2]
```diff
@@ -53,6 +53,7 @@ end_collection_event:
 imageParam_event:
     device: CCCamera
     topic: imageReadoutParameters
+    array_keys: ccdLocation
```
Comment (patrickingraham) [inline etc/conf/ccTelemetry.yaml:56]: and this...

File: start_docker_dev.sh [hunk 3]
```diff
@@ -1,5 +1,6 @@
 HSUSER=headerservice
 TAGNAME=4.1.1-5.1.0-salobj_5.10.0-hs_2.2.0
```
Comment (patrickingraham) [inline start_docker_dev.sh]: should this not be removed if you re-declare right below?

File: etc/conf/ccTelemetry.yaml (hunk 2 above)
Comment (patrickingraham) [reply etc/conf/ccTelemetry.yaml:56]: Does Tony need to review this as well?

**Review by patrickingraham — CHANGES_REQUESTED** [review]
See inline comments. 
There are comcam changes in here, is someone else reviewing those?

File: etc/conf/atTelemetry.yaml (hunk 1 above)
Comment (menanteau) [reply etc/conf/atTelemetry.yaml:55]: The key that maps to the arrays for CCD/Sensors. Here I only made `array_keys` visible at the configuration label.

File: etc/conf/ccTelemetry.yaml (hunk 2 above)
Comment (menanteau) [reply etc/conf/ccTelemetry.yaml:56]: Same. No, Tony doesn't need to review this.

File: start_docker_dev.sh (hunk 3 above)
Comment (menanteau) [reply start_docker_dev.sh]: Yes, I'll clean this one up.

**Review by patrickingraham — APPROVED** [review]
Looks good. Consider rebasing/squashing commits to remove blank line etc.

**Pull request merged**